### PR TITLE
[T3A3][T16-A2] Varun Gupta

### PR DESCRIPTION
**Pull request by varung97**
Ready for review

### CODE REVIEW

File: doc/UserGuide.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Format: `help`
  
 ### Adding a person: `add`
 Adds a person to the address book<br>
-Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...` 
+Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/BLOCK, STREET, UNIT, POSTAL_CODE [t/TAG]...`
```
Comment (PierceAndy) [inline doc/UserGuide.md]: Good job on remembering to update the documentation!

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -3,56 +3,69 @@
...
-
-    @Override
-    public int hashCode() {
-        return value.hashCode();
-    }
-
-    public boolean isPrivate() {
-        return isPrivate;
-    }
+	public static final String EXAMPLE = "";
```
Comment (PierceAndy) [inline src/seedu/addressbook/data/person/Address.java]: As previously mentioned, do pay attention to your indentation. Remember to stick to the prescribed standard, even if it's not to your personal liking.

File: src/seedu/addressbook/data/person/Address.java [hunk 3]
```diff
@@ -12,11 +12,17 @@
...
 	public static final String ADDRESS_VALIDATION_REGEX = ".+?, .+?, .+?, .+";
 
-	public final Block block;
-	public final Street street;
-	public final Unit unit;
-	public final PostalCode postalCode;
+	private final Block block;
+	private final Street street;
+	private final Unit unit;
+	private final PostalCode postalCode;
```
Comment (PierceAndy) [inline src/seedu/addressbook/data/person/Address.java]: Good job on remembering to encapsulate the class attributes, but try to do it in a separate commit next time.

**Comment by PierceAndy** [comment]
@varung97 Some comments added. Please close the PR after reading comments. Good job on updating the comments, tests, and documentations as well. Few are as thorough as you have been. Keep up the good work!

**Comment by varung97** [comment]
Thanks!
